iam-policy-autopilot-policy-generation/src/extraction/python/common/argument_extractor.rs

```diff
@@ -52,6 +52,7 @@ impl ArgumentExtractor {
                     value: Self::extract_parameter_value(&arg_text),
                     position: parameter_position,
                     type_annotation: None,
+                    struct_fields: None,
```

Comment:
Could we implement `Default` and use `..Default::default()` here, so we don't need to do these updates across the code base in the future?

If you agree, could you open an issue?